skydoc/rule_extractor.py

```diff
@@ -98,10 +99,10 @@ def _process_skylark(self, bzl_file, load_symbols):
       compiled = compile(f.read(), bzl_file, 'exec')
     global_stubs = create_stubs(SKYLARK_STUBS, load_symbols)
     env = global_stubs.copy()
-    exec(compiled) in env
+    exec(compiled, env)
```

Comment:
Is this change causing https://buildkite.com/bazel/skydoc/builds/106#ac6d3b11-ed4d-42b7-8d19-48add49cc96c?

Reply:
Oh, looks like Ubuntu 14.04 still has Python 2.6 which has additional restrictions on this.
I will try to find a workaround :)

Reply:
Cool!

Reply:
I was slightly wrong - Ubuntu 14.04 has Python 2.7.6 which still contains a bug regarding the exec statement. I found a workaround and it works fine on Python 2.7.6, Python 2.7.15 and Python 3.7 now. :)